### PR TITLE
Parmin graphing

### PR DESCRIPTION
**Pull request by parminpatel28**

### CODE REVIEW

**Review by eevanwong — APPROVED** [review]

**Comment by eevanwong** [comment]
seems good but just to be sure wait for check to complete

**Comment by shivam-sh** [comment]
I'll look into why when I get a chance

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_FIREBASE_PUBLIC_API_KEY = 
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN = 
+NEXT_PUBLIC_FIREBASE_PROJECT_ID = 
+NEXT_PUBLIC_FIREBASE_DATABASE_URL = 
+NEXT_PUBLIC_FIREBASE_MESSAGINIG_SENDER_ID =
```
Comment (shivam-sh) [inline .env.sample:5]: Can you rebase with develop & fix this typo? I think the build should work after that

Comment (parminpatel28) [reply .env.sample]: I dont have to rebase with develop since the env.sample isn't in develop branch yet.

Comment (shivam-sh) [reply .env.sample]: Nah, you do. I made workflow changes

Comment (shivam-sh) [reply .env.sample]: Wait, I lied ig, github automatically updates that nvm

**Review by shivam-sh — APPROVED** [review]

**Pull request merged**